Sources/Fuzzilli/CodeGen/CodeGeneratorWeights.swift

```diff
@@ -89,8 +89,8 @@ public let codeGeneratorWeights = [
     "AsyncArrowFunctionGenerator":              1,
     "AsyncGeneratorFunctionGenerator":          1,
     "ConstructorGenerator":                     5,
-    "SubroutineReturnGenerator":                3,
-    "YieldGenerator":                           2,
+    "SubroutineReturnGenerator":                5,
```

Comment:
Ah sorry, I meant the weights of `YieldGenerator` and `YieldEachGenerator` not the `SubroutineReturnGenerator` (that can probably stay at 3) 😅

Reply:
Got it, thanks for the correction!